doc/RELEASING.md

```diff
@@ -0,0 +1,89 @@
+# How to publish a new release of the Rust ILP Client library
```

Comment:
Nit:
Title implies this is only for Rust, when it's for the whole of c-questdb-client.